bgpd/rfapi/rfapi_rib.c

```diff
@@ -2108,10 +2108,9 @@ void rfapiRibPendingDeleteRoute(struct bgp *bgp, struct rfapi_import_table *it,
 				sl);
 
 			for (cursor = NULL,
-			    rc = skiplist_next(sl, NULL, (void **)&m,
-					       (void **)&cursor);
+			    rc = skiplist_next(sl, NULL, (void **)&m, &cursor);
```

Comment:
What about (void **)&m typecast here. This is also unnecessary here right.

Reply:
This cast is necessary. You can't implicitly cast `struct rfapi_monitors_eth **` to `void **` as they are pointers to different types. `void **` is not like `void *` in this case.